addons/binding/org.openhab.binding.foobot/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="foobot" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="device">
+        <label>Foobot</label>
+        <description>Shows the air quality data from the Foobot device.</description>
+
+        <channels>
+            <channel id="temperature" typeId="temperature" />
+            <channel id="humidity" typeId="humidity" />
+            <channel id="voc" typeId="voc" />
+            <channel id="co2" typeId="co2" />
+            <channel id="pm" typeId="pm" />
+            <channel id="gpi" typeId="gpi" />
+
+        </channels>
+
+        <config-description>
+            <parameter name="apikey" type="text" required="true">
+                <context>password</context>
+                <label>API Key</label>
+                <description>You can request your API Key from https://api.foobot.io/apidoc/index.html </description>
+            </parameter>
+
+            <parameter name="username" type="text" required="true">
+                <label>Username</label>
+                <description>The Username that you use to login to the Foobot account.</description>
+            </parameter>
+
+            <parameter name="mac" type="text" required="true">
+                <label>MAC Address</label>
+                <description>MAC address of your device. You can find it in the Foobot App under settings.</description>
+            </parameter>
+
+            <parameter name="refresh" type="integer" min="5" required="false">
+                <label>Refresh interval</label>
```

Comment:
Also please capitalize yours labels consistently, to be consistent with the rest of openHAB that should be:

`Refresh Interval` 

So start both words with a capital.